package.json

```diff
@@ -6,8 +6,8 @@
   "scripts": {
     "build": "truffle build",
     "console": "truffle console",
-    "migrate": "truffle migrate --network development",
-    "deploy": "truffle deploy --network development",
+    "migrate": "truffle migrate",
+    "migrate": "truffle deploy",
```

Comment:
Not really familiar with javascript, but is it ok to use the same name here?

Reply:
seems not😂, having duplicate `migrate` would fail the README command `yarn deploy --network production`

Reply:
fixed